README.md

```diff
@@ -43,6 +43,8 @@ Check out these [code examples](examples) in JavaScript and TypeScript to get up
 
 `twilio-node` supports credential storage in environment variables. If no credentials are provided when instantiating the Twilio client (e.g., `const client = require('twilio')();`), the values in following env vars will be used: `TWILIO_ACCOUNT_SID` and `TWILIO_AUTH_TOKEN`.
 
+Also take note that if you invoke any V2010 operations and do not set an account SID, the library will automatically use the `TWILIO_ACCOUNT_SID` value that the client was initialized with.
```

Comment:
Think this should be split out into a separate section for _Client Initialization_ with examples on how to initialize the client using the auth token, using an API key, how to set the account SID, how to use a different account SID for v2010 APIs after client initialization.